### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by origamiserviceuser**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@financial-times/o-buttons: 7.7.1</summary>

### Bug Fixes

* attach onClick handler ourselves ([a5ea55f](https://www.github.com/Financial-Times/origami/commit/a5ea55f49a47fd535253050676d5d1678e9363ab))
* separate linkbutton and button ([95ed2b3](https://www.github.com/Financial-Times/origami/commit/95ed2b31a1a778ca13df527bbf1d8fc663fad41f))
</details>
<details><summary>@financial-times/o-header: 10.1.0</summary>

### Features

* add tsx compiler to components ([e533fcc](https://www.github.com/Financial-Times/origami/commit/e533fccb9e80d758a21b52d4a176248f46d11a50))
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: components/o-header/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [10.1.0](https://www.github.com/Financial-Times/origami/compare/o-header-v10.0.1...o-header-v10.1.0) (2022-08-15)
+
+
+### Features
+
+* add tsx compiler to components ([e533fcc](https://www.github.com/Financial-Times/origami/commit/e533fccb9e80d758a21b52d4a176248f46d11a50))
```
Comment (notlee) [inline components/o-header/CHANGELOG.md:8]:
```suggestion
* Add storybook / tsx templates ([e533fcc](https://www.github.com/Financial-Times/origami/commit/e533fccb9e80d758a21b52d4a176248f46d11a50))
```

File: components/o-header/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [10.1.0](https://www.github.com/Financial-Times/origami/compare/o-header-v10.0.1...o-header-v10.1.0) (2022-08-17)
+
+
+### Features
+
+* add tsx compiler to components ([e533fcc](https://www.github.com/Financial-Times/origami/commit/e533fccb9e80d758a21b52d4a176248f46d11a50))
```
Comment (notlee) [inline components/o-header/CHANGELOG.md:8]:
```suggestion
* Add storybook / tsx templates ([e533fcc](https://www.github.com/Financial-Times/origami/commit/e533fccb9e80d758a21b52d4a176248f46d11a50))
```

File: components/o-buttons/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### [7.7.1](https://www.github.com/Financial-Times/origami/compare/o-buttons-v7.7.0...o-buttons-v7.7.1) (2022-08-17)
+
+
+### Bug Fixes
+
+* attach onClick handler ourselves ([a5ea55f](https://www.github.com/Financial-Times/origami/commit/a5ea55f49a47fd535253050676d5d1678e9363ab))
+* separate linkbutton and button ([95ed2b3](https://www.github.com/Financial-Times/origami/commit/95ed2b31a1a778ca13df527bbf1d8fc663fad41f))
```
Comment (notlee) [inline components/o-buttons/CHANGELOG.md:9]:
```suggestion
* Update onClick handler to support React's MouseEventHandler ([a5ea55f](https://www.github.com/Financial-Times/origami/commit/a5ea55f49a47fd535253050676d5d1678e9363ab))
* Fix issue with link / button type ([95ed2b3](https://www.github.com/Financial-Times/origami/commit/95ed2b31a1a778ca13df527bbf1d8fc663fad41f))
```

**Review by notlee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-buttons is at https://github.com/Financial-Times/origami/releases/tag/o-buttons-v7.7.1 :sunflower:

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-header is at https://github.com/Financial-Times/origami/releases/tag/o-header-v10.1.0 :sunflower:

**Pull request merged**